.eslintrc.js

```diff
@@ -204,6 +204,14 @@ module.exports = {
         "global-require": "off",
       },
     },
+    {
+      files: [
+        "packages/{block-scripts,block-template,blockprotocol,create-block-app}/**",
+      ],
+      rules: {
+        "unicorn/filename-case": "error",
+      },
+    },
```

Comment:
Bonus: makes sure we don’t fallback to `camelCase` in these packages. We still need to convert `mock-block-dock` and `react-block-loader` (out of scope for this PR)